docs/Getting Started/index.rst

```diff
@@ -13,7 +13,7 @@ documentation <https://pthree.org/2012/04/17/install-zfs-on-debian-gnulinux/>`__
 
    Arch Linux/index
    Debian/index
-   Fedora
```

Comment:
Let's do the same as with https://github.com/openzfs/openzfs-docs/pull/180/files#r681526008

Reply:
Thanks for the notice, fixed.